src/components/CurrencySymbolButton.js

```diff
@@ -3,24 +3,30 @@ import {TouchableOpacity} from 'react-native';
 import PropTypes from 'prop-types';
 import Text from './Text';
 import styles from '../styles/styles';
+import Tooltip from './Tooltip';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
 
 const propTypes = {
     /** Currency symbol of selected currency */
     currencySymbol: PropTypes.string.isRequired,
 
     /** Function to call when currency button is pressed */
     onCurrencyButtonPress: PropTypes.func.isRequired,
+
+    ...withLocalizePropTypes,
 };
 
 function CurrencySymbolButton(props) {
     return (
-        <TouchableOpacity onPress={props.onCurrencyButtonPress}>
-            <Text style={styles.iouAmountText}>{props.currencySymbol}</Text>
-        </TouchableOpacity>
+        <Tooltip text={props.translate('common.currency')}>
```

Comment:
As discussed in the issue just now, let's update this to use https://github.com/Expensify/App/blob/5ab92f54240381b7e2ebe78af0129d429c4d6b95/src/languages/es.js#L148 and then we can delete the new translations that are no longer needed. Thanks!